mirrord/cli/src/operator.rs

```diff
@@ -223,12 +223,19 @@ Operator License
 
     let mut sessions = Table::new();
 
-    sessions.add_row(row!["Session ID", "Target", "User", "Session Duration"]);
+    sessions.add_row(row![
+        "Session ID",
+        "Target",
+        "Namespace",
+        "User",
+        "Session Duration"
+    ]);
 
     for session in &status.sessions {
         sessions.add_row(row![
             session.id.as_deref().unwrap_or(""),
             &session.target,
+            session.namespace.as_deref().unwrap_or(""),
```

Comment:
What do you think about printing `N/A` instead? Empty field might suggest a default namespace, which is not the case

Reply:
great idea, ty